### DISCUSSION TITLE
Wrapping a field in a function to return a valid type?

### DISCUSSION THREAD

**Discussion by psmaAaron** [prisma/prisma#8549] [Q&A] [answered]
Hi everyone!

I've got a PostgreSQL DB with PostGIS installed and a bunch of tables with geometries.

Unfortunately Prisma can't handle these (yet?), but there is a function in PostGIS that converts the geometry to GeoJSON (`ST_AsGeoJSON`) that would work perfectly for my uses.

So, I was wondering if there's a way I could tell Prisma to wrap the column/field in the ` ST_AsGeoJSON` function when executing a `SELECT` from the DB.

**Comment by ryands17** [comment, marked as answer]
@psmaAaron 👋 

I think the only way to do this for now would be wrapping the column directly in a [raw query](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access).

We have a feature for supporting PostGIS so it would be great if you could add a 👍 along with your use case 🙂 
https://github.com/prisma/prisma/issues/2789

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
